src/ir/comp.rs

```diff
@@ -605,11 +605,17 @@ impl CompInfo {
                 CXCursor_UnionDecl |
                 CXCursor_ClassTemplate |
                 CXCursor_ClassDecl => {
+                    // We can find non-semantic children here, see:
+                    // https://github.com/servo/rust-bindgen/issues/482
```

Comment:
Please leave a one or two sentence description of why/when we can find non-semantic children here, and only link the issue if you think it is still relevant with those sentences.